src/pages/MapPage.js

```diff
@@ -16,14 +17,15 @@ const MapPage = () => {
       <header className="view__header">
         <h1>What's Near Me?</h1>
         {status === 'loading' ? <Spinner /> : null}
+        <SearchForm />
       </header>
       <div className="view__content">
         {status === 'error' ? <ErrorMessage error={error} /> : null}
         <Map
           defaultCenterMap={defaultCenterMap}
           centerMap={newCenterMap}
           locations={locations}
-          zoom={16}
+          zoom={14}
```

Comment:
We have the zoom level hard-coded in 2 places. Could we, should we make this a constant that’s used throughout the app?

Reply:
It could be a good idea if we add a new state `levelZoom` in our MapCenterProvider and avoid hardcoding values. We created a [Jira ticket ](https://the-collab-lab.atlassian.net/browse/TCL30-49)for fixing it. 😄 

Only as an extra comment: one change is a hardcode value, yes! I'm so sorry, but the second change it's a 'default' value if we don't pass this prop. 😄